ThingIFSDK/ThingIFSDK/Trait/TraitCommandForm.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TraitCommandForm.swift
+//  ThingIFSDK
+//
+//  Created on 2016/12/08.
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+/**
+Form of a command for trait version thing.
+
+This class contains data in order to create `Command` with
```

Comment:
If only return commandID, developers need to request `getCommand` to get the real command is the same.  Developer still be confused that I submitted with `[[String: [String: Any]]]` actions but the command returned is `[[String: Any]]`. `[[String: Any]]` is not so convenient, need to explicit convert when accessing Alias actions.